docs/src/components/NavigationFooter/NavigationFooter.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow strict */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { cssModules } from 'bpk-react-utils';
+import ChevronRightIcon from 'bpk-component-icon/lg/chevron-right';
+import { withRtlSupport } from 'bpk-component-icon';
+
+import Heading from '../Heading';
+
+import STYLES from './NavigationFooter.scss';
+
+const RtlSupportedChevron = withRtlSupport(ChevronRightIcon);
+const getClassName = cssModules(STYLES);
+
+type NavigationFooterProps = {
+  id: string,
+  title: string,
+  link: string,
+};
+
+const NavigationFooter = (props: NavigationFooterProps) => {
+  const { id, title, link } = props;
+  return (
+    <a
```

Comment:
This needs to be a react router link. The way it is right now is refreshing to whole page instead of re-rendering the content inside.